test/test_redis_connection.rb

```diff
@@ -83,6 +83,12 @@ def server_connection(*args)
           ENV.delete('RAILS_MAX_THREADS')
         end
       end
+
+      it "raises if the server pool size is too small to support other features" do
+        Sidekiq.options[:concurrency] = 6
```

Comment:
`Sidekiq.options` represents a global state (configuration), please return it back to old value on test teardown 🙇

Reply:
This is already reset in the top-level `before` block of this test file.